constants/constants.go

```diff
@@ -38,7 +39,7 @@ const ShortDockerfileFlag = "D"
 const ManifestFlag = "manifest"
 
 //ShortManifestFlag defines the shorthand manifest file
-const ShortManifestFlag = "m"
+const ShortManifestFlag = "M"
```

Comment:
Is this to differentiate between mount and manifest?

Reply:
Yes, batch and run now use both so they need to be different.